arcgis-ios-sdk-samples/Display information/Create symbol styles from web styles/CreateSymbolStylesFromWebStylesLegendTableViewController.swift

```diff
@@ -0,0 +1,47 @@
+// Copyright 2021 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+
+class CreateSymbolStylesFromWebStylesLegendTableViewController: UITableViewController {
+    /// The symbols to display in the legend table.
+    var symbols: [(name: String, symbol: AGSSymbol)]?
```

Comment:
```suggestion
    var symbols = [(name: String, symbol: AGSSymbol)]()
```
Would it be possible to initialize the array like so to avoid optional chaining?